src/components/atoms/cartMenuItem.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+	<li>
+		<a class="cart-menu__text">{{item.innerText}}</a>
+	</li>	
+</template>
+
+<script>
+	export default {
+		name: 'cartMenuItem',
+		props: ["item"]
+	}
+</script>
+
+<style lang="scss">
+.cart-menu__text {
```

Comment:
If you are using scss better structure would be
.card-item -> Parent element {
   // common style
  &__text {
   component style
  }
  &__text:hover {
  // hover style
  }
}